src/components/HomeSearchBox/index.tsx

```diff
@@ -20,23 +40,63 @@ const HomeSearchBox = () => {
 
   const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
     if (event.key === 'Enter') {
+      setActive(0)
+      setIsShow(false)
+      setSearchValue(filtered[active])
       handleSearchClick()
+    } else if (event.key === 'Up') {
+      return active === 0 ? null : setActive(active - 1)
+    } else if (event.key === 'Down') {
+      return active - 1 === filtered.length ? null : setActive(active + 1)
+    }
```

Comment:
Quick question, with this particular part of code, how does the suggestion box should behave? Currently on my local it's just moving the **text cursor** 🤔